example/spring-data-mongodb/src/test/kotlin/com/github/inflab/example/spring/data/mongodb/extension/AtlasTestConstructorExtension.kt

```diff
@@ -29,8 +33,8 @@ internal object AtlasTestConstructorExtension : ConstructorExtension {
             return null
         }
 
-        val connectionString = "mongodb+srv://$ATLAS_DOMAIN/${atlasTest.database}?retryWrites=true&w=majority"
-        val mongoTemplate = MongoTemplate(SimpleMongoClientDatabaseFactory(connectionString))
+        val connectionString = getConnectionString(atlasTest.database)
+        val mongoTemplate = getMongoTemplate(connectionString)
 
         val parameters = testConstructor.parameters.associateWith { parameter ->
             val parameterClass = parameter.type.classifier as KClass<*>
```

Comment:
If you don mind, Could you change 25, 43 lines to use `findAnnotation` extension function?

```kt
      val atlasTest = clazz.findAnnotation<AtlasTest>() ?: return null

     checkNotNull(parameterClass.findAnnotation<Repository>()) {
 ```